### PR TITLE
Init & 00

### PR DESCRIPTION
**Pull request by LCRT215**

### CODE REVIEW

**Comment by ticotheps** [comment]
Looks like you're making good progress, @LCRT215! Keep it up! 👍

File: src/03_modules.py [hunk 1]
```diff
@@ -6,26 +6,32 @@
 """
 
 import sys
+import os
 # See docs for the sys module: https://docs.python.org/3.7/library/sys.html
 
 # Print out the command line arguments in sys.argv, one per line:
 # YOUR CODE HERE
 
+print("* sys.argv:", sys.argv[0])
```
Comment (ticotheps) [inline src/03_modules.py]: You're ALMOST there with this one!

File: src/03_modules.py [hunk 2]
```diff
@@ -6,26 +6,32 @@
 """
 
 import sys
+import os
 # See docs for the sys module: https://docs.python.org/3.7/library/sys.html
 
 # Print out the command line arguments in sys.argv, one per line:
```
Comment (ticotheps) [inline src/03_modules.py]: Make sure to read the instructions VERYYYYYYY carefully, one more time! 💯

File: src/03_modules.py [hunk 3]
```diff
@@ -6,26 +6,32 @@
...
 # YOUR CODE HERE
 
+print("* sys.argv:", sys.argv[0])
+# the list of commandline arguments passed to the Python program. argv represents all the items that come along via the command line input, it's basically an array holding the command line arguments of our program. Don't forget that the counting starts at zero (0) not one (1).
+
 # Print out the OS platform you're using:
 # YOUR CODE HERE
 
+print("* OS platform name:", os.name)
```
Comment (ticotheps) [inline src/03_modules.py]: @LCRT215, check the docs again for this one! I think you may want to use "sys" here vs. "os"!  ;)

File: src/03_modules.py [hunk 4]
```diff
@@ -6,26 +6,32 @@
...
+
 # Print out the version of Python you're using:
 # YOUR CODE HERE
+print("* Python version:", sys.version)
```
Comment (ticotheps) [inline src/03_modules.py]: 👍

File: src/03_modules.py [hunk 5]
```diff
@@ -6,26 +6,32 @@
...
 
-
-import os
 # See the docs for the OS module: https://docs.python.org/3.7/library/os.html
 
 # Print the current process ID
 # YOUR CODE HERE
-
+print("* current process id:", os.getpid())
```
Comment (ticotheps) [inline src/03_modules.py]: Nice!

File: src/03_modules.py [hunk 6]
```diff
@@ -6,26 +6,32 @@
...
 # Print the current working directory (cwd):
 # YOUR CODE HERE
-
+print("*current working directory:", os.getcwd())
```
Comment (ticotheps) [inline src/03_modules.py]: You got it!

File: src/03_modules.py [hunk 7]
```diff
@@ -6,26 +6,32 @@
...
 # Print out your machine's login name
 # YOUR CODE HERE
+print("machine's login name:", os.getlogin())
```
Comment (ticotheps) [inline src/03_modules.py]: Technically, this is correct, but head back to the docs to see if there is a method that you could use to return a more SPECIFIC login name! :)

Comment (ticotheps) [inline src/03_modules.py]: Screen Shot 2020-02-13 at 12 11 32 AM

File: src/01_bignum.py [hunk 8]
```diff
@@ -1,4 +1,8 @@
 # Print out 2 to the 65536 power
 # (try doing the same thing in the JS console and see what it outputs)
 
-# YOUR CODE HERE
\ No newline at end of file
+# YOUR CODE HERE
+
+print(2**65536)
```
[the remark this answers is not in the document]
Comment (LCRT215) [reply src/01_bignum.py]: 😂

File: src/05_lists.py [hunk 9]
```diff
@@ -7,23 +7,40 @@
...
+# The append() method adds an item to the end of the list.
+
+x.append(4)
+print("- Append:", x)
 
 # Using y, change x so that it is [1, 2, 3, 4, 8, 9, 10]
-# YOUR CODE HERE
-print(x)
+
+# The extend() extends the list by adding all items of a list (passed as an argument) to the end. The extend() method only modifies the original list. It doesn't return any value.
```
Comment (ticotheps) [inline src/05_lists.py]: Loving these comments in your code that help you define the methods, @LCRT215 ! 👍

File: src/08_comprehensions.py [hunk 10]
```diff
@@ -12,13 +12,19 @@
 
 y = []
 
+for e in range(5):
```
Comment (ticotheps) [inline src/08_comprehensions.py]: @LCRT215 : This is the correct logic, but they want you to use list comprehension syntax (on line 13)!

File: src/08_comprehensions.py [hunk 11]
```diff
@@ -12,13 +12,19 @@
...
+    y.append(e+1)
+    
 print (y)
 
 # Write a list comprehension to produce the cubes of the numbers 0-9:
 # [0, 1, 8, 27, 64, 125, 216, 343, 512, 729]
 
 y = []
 
+for e in range(10):
```
Comment (ticotheps) [inline src/08_comprehensions.py]: Same with this exercise as above!

File: src/08_comprehensions.py [hunk 12]
```diff
@@ -28,6 +34,9 @@
 
 y = []
 
+for e in (a):
```
Comment (ticotheps) [inline src/08_comprehensions.py]: And this one too! :)

File: src/08_comprehensions.py (hunk 10 above)
Comment (ticotheps) [reply src/08_comprehensions.py]: Screen Shot 2020-02-19 at 9 17 57 PM
Check out the documentation here for some clues on how they want you to format it! :) https://docs.python.org/3/tutorial/datastructures.html#list-comprehensions

**Pull request merged**